src/keystore.c

```diff
@@ -234,58 +234,59 @@ static bool _verify_seed(
     return true;
 }
 
-bool keystore_encrypt_and_store_seed(
+keystore_error_t keystore_encrypt_and_store_seed(
     const uint8_t* seed,
     uint32_t seed_length,
     const char* password)
 {
     if (memory_is_initialized()) {
-        return false;
+        return KEYSTORE_ERR_MEMORY;
     }
     keystore_lock();
     if (!_validate_seed_length(seed_length)) {
-        return false;
+        return KEYSTORE_ERR_MEMORY;
```

Comment:
Shouldn't this be `KEYSTORE_ERR_SEEDSIZE`?

Reply:
Good catch, fixed!